bitbot/util.go

```diff
@@ -37,6 +37,29 @@ type ACL struct {
 	Rejected []string
 }
 
+//contains returns a bool if the string is found in the slice of strings
```

Comment:
If the comment describes return values/operation, it should indicate _all_ of them. In this case something like "contains returns the index position of a match element and bool indicating success of a match"

Reply:
Also, add a space after the `//`